.github/workflows/publish_image.yml

```diff
@@ -0,0 +1,63 @@
+# This workflow is based on example from # https://github.com/marketplace/actions/build-and-push-docker-images
+
+name: Publish Image to Volttron DockerHub
+
+on:
+  push:
+    tags:
+      - v*
+jobs:
+  build:
+    env:
+      # variables to change image name on DockerHub; here, the image name will take form volttron/<GROUP>:<TAG>
+      IMAGE_BASE_NAME: eclipsevolttron/volttron
+
+    # The strategy allows customization of the build and allows matrixing the version of os and software
+    # https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-20.04 ]
```

Comment:
8.1.1 officially supports 18.04 and python 3.6. Let's stick to that.

Reply:
Will change.